domFoundations/events,storage,attributes/lightSwitcher.js

```diff
@@ -0,0 +1,10 @@
+import { backgr } from "./initialPageRenderer.js";
+import { dateField } from "./initialPageRenderer.js";
+import { saveData } from "./storage.js";
+
+export const lightSwitcher = (check) => {
```

Comment:
the `check` is very general varname. by concreet

Reply:
Changed